samples/ei_ncs/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _ei_ncs_sample:
+
+Edge Impulse wrapper
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Edge Impulse wrapper sample demonstrates the functionality of the :ref:`ei_ncs`.
+It shows how to use the wrapper to run a custom `Edge Impulse`_ machine learning model.
+
+Overview
+********
+
+The sample performs the following operations:
+
+* Initializes the Edge Impulse wrapper.
+* Provides input features for the wrapper.
```

Comment:
I have a problem with a word `features`. In our case it really mean raw sensor data.
The EI documentation is actually a bit unclear about the naming - or my quick scrolling through it does help me resolve the strange naming.

I expect that you have following
* raw data, that is preprocessed and converted to
* (signal) features, that are passed to neural network to convert to
* result

However EI one explicitly talks about rawdata and signal features extracting and on the other occasion they talk about raw data as it would be extracted signal features. Or do they expect some preprocessing will be done with sensor data that is NOT part of the model itself?

Anyway the term features is really misleading. I recommend either use raw data or sensor data. If we for some reason need to stick with `features` let's use ei_features instead and properly document what this is.

Reply:
You could do any signal preprocessing before providing the data through the wrapper, so the input data could be raw data (probably this will be used in most cases) or a set of prepared features. Some signal preprocessing blocks are available out-of-the box in EI studio. One of them simply forwards the received input data without modifying it. The neural network could also use raw data as an input.

I think we should use the `input data` term here to be more generic, since the data could actually be either raw samples or preprocessed features.